server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/RequestHandler.java

```diff
@@ -95,16 +97,16 @@ public ErrorResponse generateInvalidArgumentError(String description) {
             .build();
     }
 
-    private Stream<JmapResponse> errorNotImplemented(JmapFieldNotSupportedException error, AuthenticatedRequest request) {
-        return Stream.of(
+    private Flux<JmapResponse> errorNotImplemented(JmapFieldNotSupportedException error, AuthenticatedRequest request) {
```

Comment:
Mono?

Reply:
Not really, a flux is expected by the caller.

Reply:
ok